bin/run

```diff
@@ -6,11 +6,6 @@ run_back_end_tests() {
     coverage run ./manage.py test
 }
 
-# TODO
-#run_front_end_tests() {
-    # ./node_modules/gulp/bin/gulp.js test
-#}
-
```

Comment:
We could probably remove this altogether, right?

Reply:
The file? We still need the tests to run for the couple tests that are left.